app/models/transaction.rb

```diff
@@ -1,3 +1,5 @@
 class Transaction < ApplicationRecord
   belongs_to :invoice
+
+  scope :successful, -> { where(result: 'success') }
```

Comment:
@tmikeschu I was thinking of adding this scope for the logic I was working on as well. Thank you!